src/events/client/messageCreate.ts

```diff
@@ -62,9 +62,7 @@ export default async function messageCreateHandler(message: Eris.Message) {
     } else if (state.gameSessions[message.guildID]?.gameRound) {
         if (areUserAndBotInSameVoiceChannel(message)) {
             const gameSession = state.gameSessions[message.guildID];
-            if (!gameSession.isMultipleChoiceMode()) {
-                gameSession.guessSong(MessageContext.fromMessage(message), message.content);
-            }
+            gameSession.guessSong(MessageContext.fromMessage(message), message.content);
```

Comment:
We should still block regular guesses when in multiple choice mode

Reply:
There's no clean way to do this when switching between mc and typing mid-game. Also, typing is objectively a handicap